config/params/mainnet_config.go

```diff
@@ -203,12 +202,6 @@ var mainnetBeaconConfig = &BeaconChainConfig{
 	BellatrixForkEpoch:   math.MaxUint64,
 	ShardingForkVersion:  []byte{3, 0, 0, 0},
 	ShardingForkEpoch:    math.MaxUint64,
-	ForkVersionSchedule: map[[4]byte]types.Epoch{
-		{0, 0, 0, 0}: genesisForkEpoch,
-		{1, 0, 0, 0}: mainnetAltairForkEpoch,
-		{2, 0, 0, 0}: mainnetBellatrixForkEpoch,
-		// Any further forks must be specified here by their epoch number.
-	},
```

Comment:
See the new `init()` function in `config/params`. Rather than duplicate this information here in the declaration, the code nows uses the named fork/epoch field params to build the `ForkVersionSchedule` at init time. The benefit of this is that we can guarantee the version identifiers are globally unique and use them for fork+config detection.